lib/matplotlib/table.py

```diff
@@ -105,14 +105,38 @@ def get_fontsize(self):
         """Return the cell fontsize."""
         return self._text.get_fontsize()
 
-    def auto_set_font_size(self, renderer):
-        """Shrink font size until the text fits into the cell width."""
+    def auto_set_font_size(self, renderer, grow=False):
+        """Adjust font size until the text fits.
+
+        Parameters
+        ----------
+        renderer:  renderer object
```

Comment:
This has to be numpydoc style. Something like this will do:

~~~
        Parameters
        ----------
        renderer : Renderer
        grow : bool
            Whether the font size should also be increased if there is space
            available.
~~~

See also https://matplotlib.org/devel/documenting_mpl.html?highlight=writing#writing-docstrings